cryosparc/dtype.py

```diff
@@ -3,41 +3,17 @@
 """
 
 import json
-from typing import TYPE_CHECKING, Dict, List, Literal, Optional, Sequence, Tuple, Type, TypedDict, Union
+from typing import TYPE_CHECKING, Dict, List, Literal, Optional, Sequence, Type, TypedDict, Union
 
 import numpy as n
 
 from .core import Data, DsetType
+from .errors import DatasetLoadError
+from .spec import DType, Field
 
 if TYPE_CHECKING:
     from numpy.typing import DTypeLike, NDArray
 
-Shape = Tuple[int, ...]
```

Comment:
Moved to spec.py to avoid unnecessarily importing the native core module when only these simple types are required.